src/go/k8s/config/manager/kustomization.yaml

```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: vectorized/redpanda-operator
-  newName: vectorized/redpanda-operator
-  newTag: latest
+  newName: localhost/redpanda-operator
+  newTag: dev
```

Comment:
I'm not sure if this change is correct.

Reply:
I'll revert it. I changed it so I could test locally because if I don't, the image doesn't exist.